youbora/package.json

```diff
@@ -30,10 +30,10 @@
   ],
   "scripts": {
     "typescript": "tsc --noEmit",
-    "lint": "eslint \"**/*.{ts,tsx}\"",
-    "prepare": "npm run manifest && bob build",
+    "build": "npm run manifest && bob build",
+    "prepare": "npm run build",
     "manifest": "youbora-util manifest",
```

Comment:
This script seems to be regenerating `youbora/manifest.json` on every `npm install`. This writes the current date to `"built"` ([source](https://github.com/THEOplayer/react-native-theoplayer-analytics/blob/b8e7acfb0c0dc69a95bd881ff23afce42627c464/youbora/manifest.json#L7)), which [pollutes the Git history a bit](https://github.com/THEOplayer/react-native-theoplayer-analytics/commits/main/youbora/manifest.json).

* Is `manifest.json` supposed to be an auto-generated file? If so, it should be in `.gitignore`.
* Is `manifest.json` supposed to be included in the [npm package](https://www.npmjs.com/package/@theoplayer/react-native-analytics-youbora)? If so, it should be listed in `package.json` under `"files"`.

Reply:
2x correct. It should be part of the packed files indeed.
The same for agama, which also has a generated manifest.